fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -203,11 +205,17 @@ private fun ReviewGradeCard(
     reviewGrade: ReviewQualityCheckState.Grade,
     modifier: Modifier = Modifier,
 ) {
-    ReviewQualityCheckCard(modifier = modifier.semantics(mergeDescendants = true) {}) {
+    ReviewQualityCheckCard(modifier) {
+        val title = stringResource(R.string.review_quality_check_grade_title)
```

Comment:
💡 We can inline this in the parameter, 

same for all the changes that are reading`stringResource`